modular_skyrat/modules/customization/modules/client/preferences.dm

```diff
@@ -270,6 +270,24 @@ GLOBAL_LIST_INIT(food, list(
 
 #define APPEARANCE_CATEGORY_COLUMN "<td valign='top' width='14%'>"
 #define MAX_MUTANT_ROWS 4
+#define FLAVORTEXT_JOIN_MINIMUM 150
+
+/datum/preferences/proc/check_flavor_text(inform_client = TRUE)
+	if(!features["flavor_text"])
+		if(inform_client)
+			to_chat(parent, "<span class='userdanger'>You must have a flavor text!</span>")
+		if(check_rights(R_ADMIN))
+			message_admins("[usr] is bypassing the flavor text requirements.")
```

Comment:
Does this throw an alert to admins saying "Do you wish to bypass flavour text requirements?" or not, cause if not, it probably should

Reply:
> Does this throw an alert to admins saying "Do you wish to bypass flavour text requirements?" or not, cause if not, it probably should

Yes, we typically want to surpass it when it comes for using quick characters for events, and some admins may forget about the flavor text limit.